src/App.js

```diff
@@ -45,21 +48,21 @@ function App() {
 
         {/* Main-Left */}
         <div className="col-span-6 row-span-4 row-start-2">
-          <Main state="left" />
+        <Main state="left"/>
         </div>
 
         {/* Main-Right Element */}
         <div className="col-span-3 row-span-3">
-          <Compost />
+        <Anim/>
```

Comment:
Might be worth having this being named more descriptively? Something along the lines of `AnimatedText` or maybe something like `CompanyPost`?